lancej1022/my-portfolio
# Added basic testing setup

## Description:
Application did not have any testing besides the static testing provided by TypeScript

## Changes I Made:
* Added husky and lintrc to help automate testing whenever code is commited
* Added jest and testing-library/react 
* Began writing some basic unit tests

## How to Test:
1. Run unit tests using npm run test
2. attempt to commit code and husky should throw errors if any are present

## Dev checklist
- [x] I have reviewed my own work
- [x] I have reviewed relevant browser/server logs
- [x] I have written appropriate tests
- [x] I have checked-in only the relevant files
- [x] I have added only appropriate documentation
- [x] I have run my code
